pkg/k8s/secrets/mocks/mocks.go

```diff
@@ -33,30 +33,30 @@ import (
 	reflect "reflect"
 )
 
-// MockSecretHelper is a mock of SecretHelper interface
+// MockSecretHelper is a mock of SecretHelper interface.
```

Comment:
Mocks are being generated since ever without a trailing period. When I generate the mocks locally it is also without period.
-> something seems to be different on your machine.

Although verification now can handle this, we should not switch this formatting back and forth when you generate or I generate. Seems we still need to investigate the reason.

Reply:
Found the change https://github.com/golang/mock/pull/414

Reply:
Prepared a fix: https://github.com/SAP/stewardci-core/pull/101